scripts/template/src/patternfly/{folder}/{name}/examples/{name}.md

```diff
@@ -1,8 +1,7 @@
 ---
-title: '{{nameCapitalized}}'
-section: beta
+title: '{name}'
```

Comment:
With a component `FooBar`, this creates `FooBar` as the title. Can that be `Foo bar`?

Reply:
@redallen can we fix this one?

Reply:
Whoops I missed this in my first pass. Fixed.